Shopify/cli
# ml/added handle generation for Flow extensions on import

### WHY are these changes introduced?

We are introducing a `handle` property for Flow extensions.

### WHAT is this pull request doing?

Generating a `handle` property for Flow extension when they're being generated via the `import-flow-legacy-extensions` command. The handle is generated using `hyphenate` which is the same function we're using the generate the folder name.

### How to test your changes?

Pull this PR, create a Flow extension in the Partner's Dashboard and import that action locally in your app. Look at the TOML file, it should have a `handle` property in it which should match the folder name.
